### ISSUE TITLE
[Bugfix - 1.7.10] MineTweaker/CraftTweaker NEI Bug

### ISSUE THREAD

**Issue by UnlimatedStone9** [einsteinsci/betterbeginnings#158]
Hey there, I am aware that 1.7.10 is quite an outdated version, however I'm going to chance my arm anyways and hope it's being maintained at least a little bit, I have this issue regarding NEI and MineTweaker/CraftTweaker support
In its simplest form the issue is, if I create a custom ore dictionary using MineTweaker/CraftTweaker and then attempt to use that ore dictionary in the recipe script for one of BetterBeginnings "Machines" it when a world is loaded, presents a bug in NEI, that custom ore dictionary appears as just the vanilla fire item, granted when I use the command /mt reload it fixes it immediately after however this is not convenient for modpacks, I cannot have players reload the scripts every time they start the game, not to mention other mods have issues with reloading scripts, like registering duplicate recipes in NEI or NEI itself hiding things and ignoring metadata

Basically BetterBeginnings has no support for custom ore dictionaries without having to reload the scripts using a command, which is incredibly inconvenient

Thanks!         -UnlimatedStone9

**Comment by UnlimatedStone9** [comment]
It's also not just an issue with NEI, the recipes themselves do not work, neither with minecraft:fire or the chosen custom ore dictionary

This appears to occur for the advanced crafting table however it might also occur for other "Machines"

**Comment by UnlimatedStone9** [comment]
![image](https://user-images.githubusercontent.com/40124890/52179738-d5ccd900-27d5-11e9-99d8-5e79c54806ab.png)

**Comment by UnlimatedStone9** [comment]
This could in fact be the result of the scripts file "Ore Dictionary" being loaded after the scripts file "BetterBeginnings Advanced Crafting Table" because of it's alphabetical order

I'll check on this

**Comment by UnlimatedStone9** [comment]
Ok, this appears to be a screw up on my part, loading the ore dictionaries after the scripts, my mistake
If I experience any more issues with the mod I'll open this again
